.circleci/config.yml

```
@@ -784,7 +784,7 @@ jobs:
             for C in ./contracts/*/
             do
               echo "Generating schema for $C ..."
-              (cd $C && cargo schema --locked && tar -zcf ~/project/schemas/$(basename $(pwd))_schema.tar.gz ./schema)
+              (cd $C && cargo schema --locked && mv ./schema/$(basename $(pwd)).json ~/project/schemas/)
```

```suggestion
              (cd $C && cargo schema --locked && mv ./schema/*.json ~/project/schemas/)
```

We don't need to care how it is named here, right?

---

We don't. Just thought in case something goes wrong, I'd rather risk uploading nothing than random things.